base/dict.jl

```diff
@@ -68,12 +68,80 @@ Dict{String,Int64} with 2 entries:
 
 Alternatively, a sequence of pair arguments may be passed.
 
-```jldoctest
-julia> Dict("A"=>1, "B"=>2)
+```jldoctest dictexamples
+julia> d = Dict("A"=>1, "B"=>2)
 Dict{String,Int64} with 2 entries:
   "B" => 2
   "A" => 1
 ```
+
+After creation of a Dict you can use `d[x]` for getting the values and `d[x]=y` for changing the values.
+```jldoctest dictexamples
+julia> d["A"]
+1
+
+julia> d["B"] = 3
+3
+```
+
+# Extended help
+
+You can use [Generator](https://docs.julialang.org/en/v1/manual/arrays/#Generator-Expressions-1) and [Comprehension][https://docs.julialang.org/en/v1/manual/arrays/#Comprehensions-1] syntax to create dictionaries:
```

Comment:
This seems like a restatment of the first part of this docstring:

> Given a single iterable argument, constructs a [`Dict`](@ref) whose key-value pairs
>  are taken from 2-tuples `(key,value)` generated by the argument.

Reply:
No, it is not. `Dict(x => x^2 for x = 0:1:4)` is different from `Dict([x => x^2 for x = 0:1:4])`

Reply:
Also, now that I am thinking, the location of the 1st and 2nd examples should be changed. The 2nd example is `2X` faster since it directly makes the `Dict`.
```julia

# Examples
```jldoctest dictexamples
julia> Dict([("A", 1), ("B", 2)])
Dict{String,Int64} with 2 entries:
  "B" => 2
  "A" => 1
```

Alternatively, a sequence of pair arguments may be passed.

```jldoctest dictexamples
julia> d = Dict("A"=>1, "B"=>2)
Dict{String,Int64} with 2 entries:
  "B" => 2
  "A" => 1
```

```